### DISCUSSION TITLE
I made Vite template that is debuggable in VSCode.

### DISCUSSION THREAD

**Discussion by tenpaMk2** [excaliburjs/Excalibur#2696] [Show and tell]
I have created a Vite template that can be debugged in VSCode❗

I know there is already a [Vite template](https://github.com/excaliburjs/template-ts-vite) available, but my template has the following new features

## VSCode debuggable

![debug](https://github.com/excaliburjs/Excalibur/assets/8945311/c8b0c6a2-97a4-4afc-9188-fccf90275018)

I think it is a bit complicated to be able to debug with VSCode. This is because the configuration changes depending on each build environment (Vite, Parcel2, WebPack, etcetc...) .

No more wasting time❗
I have already prepared `launch.json` for you.

## Tiled plugin support

I think it is also complicated to use Tiled plug-ins.
This is because modern build environments (especially Vite) append hash to file names.
This is a trap.
The Tiled plugin can not handle hashed filenames well.

No more wasting time❗
I have already prepared suitable directory structures.
The key is to use `public` directory for assets used by Tiled plugin.

**Comment by eonarheim** [comment]
@tenpaMk2 Thanks! This is so awesome!

I'm definitely trying this out as soon as I can!
